lightcycle-lib/src/main/java/com/soundcloud/lightcycle/FragmentLightCycleDispatcher.java

```diff
@@ -32,20 +34,27 @@ public void bind(FragmentLightCycle<T> lightCycle) {
 
     @Override
     public void onAttach(T fragment, Activity activity) {
-        LightCycles.bind(this);
+        bindIfNecessary();
         for (FragmentLightCycle<T> component : fragmentLightCycles) {
             component.onAttach(fragment, activity);
         }
     }
 
     @Override
     public void onAttach(T fragment, Context context) {
-        LightCycles.bind(this);
+        bindIfNecessary();
         for (FragmentLightCycle<T> component : fragmentLightCycles) {
             component.onAttach(fragment, context);
         }
     }
 
+    private void bindIfNecessary() {
```

Comment:
Since the purpose of this pull request is to prevent multiple call to `LightCycles.bind()` do you think we should test this function too?  
I think this can be done with RoboElectric's [FragmentController](https://github.com/soundcloud/lightcycle/blob/master/lightcycle-integration-test/src/test/java/com/soundcloud/lightcycle/integration_test/FragmentLoggerTest.java) (Like we did in integration test) and Mockito's [verify()](http://stackoverflow.com/questions/27787487/java-verify-void-method-calls-n-times-with-mockito) if you think it's necessary to test this function.